src/types.ts

```diff
@@ -27,7 +27,7 @@ export interface Indexable {
  *
  */
 export interface GitLabQuery extends Indexable, DataQuery, ProjectOptions {
-  options?: CommitsOptions;
+  options?: CommitsOptions | IssuesOptions;
```

Comment:
```suggestion
  options?: CommitsOptions;
```
I don't think it's needed.